mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -8257,8 +5811,7 @@
 													"});",
 													"",
 													"pm.test(\"Validate schema\", function () {",
-													"     tv4.addSchema(\"JSONAPI.json\", JSON.parse(pm.variables.get(\"schema_jsonapi_content\")));",
-													"    pm.expect(tv4.validate(response, JSON.parse(pm.environment.get(\"schema_jsonapi_content\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+													"    pm.expect(tv4.validate(response, JSON.parse(pm.environment.get(\"schema_jsonapiError\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
```

Comment:
@natali-zaitseva -- just to understand error schema name change to schema_jsonapiError. Were prior tests simply checking schema_jsonapi_content and not being specific for error response?

Reply:
yes, Carole. We were just checking general JSON api schema but not our specific error response.

Reply:
ok thanks for clarification @natali-zaitseva